tests/testthat/test-geom-ref-line.R

```diff
@@ -0,0 +1,24 @@
+context("geom-ref-line")
+
+test_that("can generate typical outputs and error message", {
+  sim <- tibble::tribble(
+           ~x,          ~y,       ~resid,
```

Comment:
This is indented too far

Reply:
Got rid of this dataset (along with the test) because the test was duplicating code inside the function